src/checkstyle/checkstyle.xml

```diff
@@ -15,19 +13,20 @@
 		<property name="packageInfoHeaderType" value="none"/>
 	</module>
 	<module name="com.puppycrawl.tools.checkstyle.checks.NewlineAtEndOfFileCheck"/>
-	<module name="JavadocPackage" /><!-- package-info.java -->
-
+	<module name="JavadocPackage"/> <!-- package-info.java -->
 	<!-- TreeWalker Checks -->
 	<module name="TreeWalker">
-
+		<!-- Imports -->
+		<module name="AvoidStarImport"/>
+		<module name="UnusedImports"/> <!-- review: needs to be explicit activated -->
```

Comment:
needs to be explicit activated, so the root configuration does not seem to work.